cities.json

```diff
@@ -279,305 +279,1761 @@
+  {"id": "2125304", "name": "تويزكي", "state_id": "21253"},
+  {"id": "2125305", "name": "عوينة يغمان", "state_id": "21253"},
+  {"id": "2125306", "name": "المحبس", "state_id": "21253"},
+  {"id": "2125307", "name": "لبويرات", "state_id": "21253"},
+
+  {"id": "2125401", "name": "كلميم", "state_id": "21254"},
+  {"id": "2125402", "name": "بويزكارن", "state_id": "21254"},
+  {"id": "2125403", "name": "افران الأطلس الصغير", "state_id": "21254"},
+  {"id": "2125404", "name": "تاغجيجت", "state_id": "21254"},
+  {"id": "2125405", "name": "تمولاي", "state_id": "21254"},
+  {"id": "2125406", "name": "تڭانت", "state_id": "21254"},
+  {"id": "2125407", "name": "أداي", "state_id": "21254"},
+  {"id": "2125408", "name": "أمتدي", "state_id": "21254"},
+  {"id": "2125409", "name": "آيت بوفلن", "state_id": "21254"},
+  {"id": "2125410", "name": "فاصك", "state_id": "21254"},
+  {"id": "2125411", "name": "اسرير", "state_id": "21254"},
+  {"id": "2125412", "name": "افركات", "state_id": "21254"},
+  {"id": "2125413", "name": "تيڭليت", "state_id": "21254"},
+  {"id": "2125414", "name": "اباينو", "state_id": "21254"},
+  {"id": "2125415", "name": "لقصابي تاڭوست", "state_id": "21254"},
+  {"id": "2125416", "name": "رأس اومليل", "state_id": "21254"},
+  {"id": "2125417", "name": "الشاطئ الأبيض", "state_id": "21254"},
+  {"id": "2125418", "name": "تلوين اساكا", "state_id": "21254"},
+  {"id": "2125419", "name": "تارڭا وساي", "state_id": "21254"},
+  {"id": "2125420", "name": "لبيار", "state_id": "21254"},
+
+  {"id": "2125501","name": "طانطان","state_id": "21255"},
+  {"id": "2125502","name": "الوطية","state_id": "21255"},
+  {"id": "2125503","name": "امسيد","state_id": "21255"},
+  {"id": "2125504","name": "تيلمزون","state_id": "21255"},
+  {"id": "2125505","name": "أبطاح","state_id": "21255"},
+  {"id": "2125506","name": "بن خليل","state_id": "21255"},
+  {"id": "2125507","name": "شبيكة","state_id": "21255"},
+
+  {"id": "2125601","name": "السمارة","state_id": "21256"},
+  {"id": "2125602","name": "حوزة","state_id": "21256"},
+  {"id": "2125603","name": "أمكالة","state_id": "21256"},
+  {"id": "2125604","name": "سيدي أحمد العروسي","state_id": "21256"},
+  {"id": "2125605","name": "الجديرية","state_id": "21256"},
+  {"id": "2125606","name": "تفاريتي","state_id": "21256"},
+
+  {"id": "2125701","name": "بوجدور","state_id": "21257"},
+  {"id": "2125702","name": "كولتة زمور","state_id": "21257"},
+  {"id": "2125703","name": "الجريفية","state_id": "21257"},
+  {"id": "2125704","name": "لمسيد","state_id": "21257"},
+
+  {"id": "2125801","name": "العيون","state_id": "21258"},
+  {"id": "2125802","name": "المرسى","state_id": "21258"},
+  {"id": "2125803","name": "بوكراع","state_id": "21258"},
+  {"id": "2125804","name": "الدشيرة","state_id": "21258"},
+  {"id": "2125805","name": "فم الواد","state_id": "21258"},
+
+  {"id": "2125901", "name": "طرفاية", "state_id": "21259"},
+  {"id": "2125902","name": "طرفاية (البلدية)","state_id": "21259"},
```

Comment:
Not sure what exactly to do here! If I removed the first two rows it will be 4 not 5.

Also, according to the [link](https://ar.wikipedia.org/wiki/%D8%A5%D9%82%D9%84%D9%8A%D9%85_%D8%B7%D8%B1%D9%81%D8%A7%D9%8A%D8%A9) '2125902' is the name for the civil city 
hence only 2125901 should be removed.
Do you agree?